scripts/mkimage

```diff
@@ -263,6 +263,14 @@ EOF
     mcopy $RELEASE_DIR/target/KERNEL.md5 "::/$KERNEL_NAME.md5"
     mcopy $RELEASE_DIR/target/SYSTEM.md5 ::/SYSTEM.md5
 
+    # compile and copy amlogic autoscript files to part1
+    for src in $INSTALL_SRC_DIR/*autoscript.src ; do
+      if [ -e "$src" ] ; then
+        $TOOLCHAIN/bin/mkimage -A $TARGET_KERNEL_ARCH -O linux -T script -C none -d "$src" "$LE_TMP/$(basename $src .src)" > /dev/null
```

Comment:
IMHO it would be cleaner to move the u-boot mkimage calls out of scripts/mkimage, probably to the bootloader package.